CP05MOAS-GL379/CP05MOAS-GL379_R00004_ingest.csv

```diff
@@ -1,7 +1,7 @@
-parser,filename_mask,reference_designator,data_source,status,notes
-#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-00-ENG000000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
-#mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/dvl/*.PD0,CP05MOAS-GL379-01-ADCPAM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
-#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-02-FLORTM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
-#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-03-CTDGVM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
-#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-04-DOSTAM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
-#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-05-PARADM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
+parser,filename_mask,reference_designator,data_source,status,notes
+#mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged/cp_*.mrg,CP05MOAS-GL379-00-ENG000000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
+#mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/dvl/*.PD0,CP05MOAS-GL379-01-ADCPAM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged/cp_*.mrg,CP05MOAS-GL379-02-FLORTM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged/cp_*.mrg,CP05MOAS-GL379-03-CTDGVM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged/cp_*.mrg,CP05MOAS-GL379-04-DOSTAM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00004/merged/cp_*.mrg,CP05MOAS-GL379-05-PARADM000,recovered_host,Not Expected,Glider leaked and was recovered within 12 hours
```

Comment:
The path was not correct for a recovered glider dataset so I updated them even though these data are not available